CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ StackOverflow is a much better place to ask questions since:
 * questions and answers stay available for public viewing so your question / answer might help someone else
 * SO voting system assures that the best answers are prominently visible.
 
-To save your and our time we will be systematically closing all the issues that are request for general support and redirecting people to StackOverflow. 
+To save your and our time we will be systematically closing all the issues that are request for general support and redirecting people to StackOverflow.
```

Comment:
What do you think about changing `To save your and our time` to `To save everyone's time`?

Please change `request` to `requests`.